logos/betking/betking.svg

```diff
@@ -0,0 +1 @@
m+56Wia5KoxigdgE9f64GzK zNPRmE18+rc8ce+eQZRGRBRxNo1iGJoFjv9XccoafvbxmNc4KhiNqo98+GNf9s7PvG3/+JefxF4x nKKNbe3rCBUmsoacbQCb5bWXKqBqpWlCJQyT1dFUnipvKp9UTVo1aRNcFEeUgRikQSNhvlZ9NwrC yWjsnAWFduPEaONkNRrGcuRDVVXTs/fv3r2UNOVxa1nZeAleQgiBiERERJIkG26Np00gm30ZYZXu hgD/Dr4GcFCNclq3j4iJFaIzc/RtDSzMZH12ts//LQi4Jvi5rR7JzGWehElUiLfKMBw1edpFiD7U xqqqgqn2LcH1u4vV5qibtXlaQkNUUhKlmYqBACJn7ifijtFAT5NWt8Fzxub8zGEhUWBujAqr4K/g hctc9QAzU3mzHSvnZBeyBMALbr3t+Lvf+xHBnWwA8ryhxDRXW5jT12ceQJYx6MH7xrmUuVNWomUD Z1xq2tCA3XhYZd1FWwxiWwMRziBJ2CbMBhCSyCDLxlkmhihUWAGB+cpoxTOC6uwVzQkWsy7tfPAJ SNLJwYRp1d21+MLvefb/+l8/9finPC76MobJZfe51wUHz4mtt9DMWcuz9Dl0urmvtZzErL90/U23 qEIbD1ioZTVGeFahREZg3mlj3bPgKLAuJeM0wswmPzLf7RQ6fZXgU1bmO/jXQYhEJXg6tzg3Dci7 iDSwesnyTgOraeFbceDEaRvGdaw4taVvFCaJvcRzoqsajhDHKAQmNYFt41xsfMXG4E7TiFOMzW1J 4tM8dYqTyQgA1EYNJnGqucTUJd26jcYlVTNnYhCJIkSEuE33MspGLamFWkMABeMQAU6QdBb/4nVv +8KXtgCowLcBUUgB31Lw80VItVBWsABx+2ZIrYIlJtzxvgdZ5v7u0NZNO3ROjNiE+9SwFRj2cA3C lBxJLGM7WuxobktqNqXaJKkhkcgY2/EKA9e0U3AzWzCg017W20GZRSkIVMkAYIKzRih4ovG0HvQX J8cO/cSPfcdLv/8RD3/45bFe7/WwNTyRd5aiz1JitBMSb1ldIpPJqLALabpnJPa2k6ukcMaROJbU xsQIze1mDaudKZPe6W0Hdwc8LWvf+BjnzkjbDYnTkUm2CdYzT/OdY/1VwSQOmios1AgDEKPCCmPI K6LJPTJCFoON3rqk0CgSfJYlvq0TtSmxtsNvePhVzljncpsUGqkZlm0TOp2+xDM4B7MCbS5fdUZB RFA6tcwmgPR6nbLCpJymmW19hckkLzplOU3TpG3rpSUwI4qKBiDqKSIZzqwIAUOANr6uQ2wCbj60 etttG9d+/nYfQIwksXPbehGw0dafWgY8dZETCBBi9FHgnKnaBkigLCGAYppTiJX33rI1ZKIPsSnh azirTcUGe1a6a0duvOzSc37wJc8vUmgs88IS0dZwaggAUpd8+QbfGSWpYPbuSEFtQAgg0iwxIj6x 6odrP/iC51q0Ahw5dJicaavjnazxZU1iWcmxIbSgNlIDQ165EW6Mq4hFhLePlNFgRc1ME+2OS5k7 uEfATdMgBGz7P+9oy3xdkBRhhfxuCn1oCk2UWEmVWzHSSDCuaHyq6IsueD8wWDRi4duONVpXBiG1 0DZ842OfCE9SQxqXYBG0oL7reAX1Nstou8CZJy4UZ2KBZ4yoBAiCGKT20hQ58tylmYm+LPb2q8ma TcRwo2inJQAYVqEZEZTMzENZz/hFFHxdAohqbGLI4od+6KfJLL7ur97SBDQRdYhgG0VNkoGYnDvz qGw7MgsAZYoEOBZpbQo4AVWwvmmGZGLecWpkUk6CxLRTJEUXPqQuTcmcuPXWxV39n/yJ//pzP/+Y hzzsyhjHbTNs6nGnmzIQW0jLEIO5gtgpV7RTB4yChNnCjLWwFq2v6+H6csc4P7Lt5uMf/oD9u5NP fvSWN/zl3+ZsC7N1wVkJ2g2HGMPMUDqCGk8aUlNxaI0EZwNzSzFQo6YinhqaWlRp9FlQEwKFyHfS ddSdodbdBhMRjLHW7gSsewLEkrKk80RICcqREEkji1dhV6xublW+dZmNLG0MzFaDEoQpitYgTxwP HtyfOoBUmlaJOv2BEnwMMHTqVLzjSO40dHs8pwDAlu1czpx588QJ5wypgtQZDcEP+t3RaMYUZygT mM+MVphHK5C6LKvb1iapAu/9wOETq+Xesy5++1ve84evfBMM2JpWAGNAiCHoqSyN5l8YYDCBQYYY PrbgaGwEapAHtSY1XsrKT5Vj0nE2tU3btKtrCG072fLl1sru5Zf//M895tH9D3xgdM01VyepMRbG ar/b9Q0YYDtrV+npfPPUehBEECzPukvqG4ggTcxgV1dlNF2/7Wnf9PhvevLFKvjAhz85nkgn7+1Z NJeev+jEO9jgTdBEhRmGRZ0SQgPAmkJREKWqhFnWRXFOsFBjhO2OoPs9DTbGgPm0zdeOa87XAVYQ AlEDnoJLUFSCECKRshHiSHzDLTef3LqdsgnSTa8b1lqoadtgExtRRaptKk3cUlsmHY88+GY9mmlD 44muI/cyy6FOuYXAzG+6vQ03v1lgJnCeMhUANjcmu3afzbDTza0kyeuyMeDYyGI/nT0XoQBSUiY9 MwmQWfSKAGzqFaMS//N//a5xS2Xtusvn/tIvv+LQsRgAYUzbGAB2DBPBAXx6WjhrJ/FcIQdpZsAh aolYAS1im+UWpCE2xEKkPjRsTG/vyqBjBxkO7Fn89V95+fO+8/IbbsD/+B8vm1R1iJq4DMCRI0e8 h3UAAXF7VRqBcCpmBcAbiCIAoQ3TJAUbbI1ObI2On9i84ZL7nfPrv/3jMPj8zdXvv/IvOov7Vjc2 u1ncu5xbgUUekQu6MeQmpp3GDhopWimicW1ufCehHqKdteoCI7ANyALyiDTO/Xv0jrcd3H2wqkIk xhhjPBWtZtnWDu4y5plJAFXgatZgUoKyEjNgFOktNx89eWIDgD
```

Comment:
Hi, @Farelz Thanks for your contribution. 
However, there seems to be an issue with the SVG file; it contains a PNG image. i.e` xlink:href="data:image/png;`
1. Do re-edit the SVG to make sure its 100% vector.
2. Go through the [**contribution.md**](https://github.com/PaystackHQ/nigerialogos/blob/master/contributing.md) on how to optimize the SVG.